src/targets/chrome/docker.js

```diff
@@ -40,13 +41,41 @@ const waitOnCDPAvailable = port =>
     );
   });
 
+const getNetworkHost = async dockerId => {
+  let host = '127.0.0.1';
+
+  // https://tuhrig.de/how-to-know-you-are-inside-a-docker-container/
+  const runningInsideDocker = (await fsExtra.exists('/proc/1/cgroup')) &&
+    /docker/.test(await fsExtra.readFile('/proc/1/cgroup', 'utf8'));
+
+  // If we are running inside a docker container, our spawned docker chrome instance will be a sibling on the default
+  // bridge, which means we can talk directly to it via its IP address.
+  if (runningInsideDocker) {
+    const { code, stdout } = await execa('docker', [
+      'inspect',
+      '-f',
+      '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}',
+      dockerId,
+    ]);
+
+    if (code !== 0) {
+      throw new Error('unable to determine ip of docker container');
```

Comment:
Sorry for being anal, but can you upper case the first letter and `IP` in the error message?